codility_training/lessons.lesson06.Sorting.MaxProductOfThree/khw970421.js

```diff
@@ -0,0 +1,19 @@
+function solution(A) {
+    A.sort((a,b)=>b-a);
+    if(A[0]*A[1]*A[2]==0){
+        return A[A.length-1]*A[A.length-2]*A[0];
+    }
+    else if(A[A.length-1]*A[A.length-2]*A[0]==0){
```

Comment:
1. else 필요 없음
2. Array.length는 변수로 저장

```suggestion
    A.sort((a,b)=>b-a);
    constl en = A.length;
    if(A[0]*A[1]*A[2]==0){
        return A[len-1]*A[len-2]*A[0];
    }
    if(A[len-1]*A[len-2]*A[0]==0){
```